src/api/style.js

```diff
@@ -0,0 +1,287 @@
+import * as _ from 'lodash';
```

Comment:
we should review this file, maybe we can split it in smaller filers

Reply:
Yes, of course, in other PR we should refactor this file and its interaction with the Layer and the Renderer: private methods called out of the class, on-the-fly properties. There are also some tests ignored because of the `parent` and `notify` addition in the StyleSpec properties.

This file is expected to grow with new properties. Instead of updating always the same file we should follow a method to allow extension with new files.

Reply:
Yes, please, don't mix refactors with features and bug fixing, the diff is already huge.